### PR TITLE
8244402: Lanai - Motif L&F - Non selected Radio button is barely rendered on non-retina display

### PR DESCRIPTION
**Pull request by dekonoplyov**
Implemented DrawPixel

---------
### Progress
- [x] Change must not contain extraneous whitespace

### Issue
 * [JDK-8244402](https://bugs.openjdk.java.net/browse/JDK-8244402): Lanai - Motif L&F - Non selected Radio button is barely rendered on non-retina display

### Reviewers
 * Ajit Ghaisas ([aghaisas](@aghaisas) - Committer)

### Download
`$ git fetch https://git.openjdk.java.net/lanai pull/67/head:pull/67`
`$ git checkout pull/67`

### CODE REVIEW

File: src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLRenderer.m [hunk 1]
```diff
@@ -85,6 +85,28 @@ void MTLRenderer_DrawLine(MTLContext *mtlc, BMTLSDOps * dstOps, jint x1, jint y1
...
+
+    id<MTLRenderCommandEncoder> mtlEncoder = [mtlc.encoderManager getRenderEncoder:dstOps];
+    if (mtlEncoder == nil)
+        return;
+
+    // Translate each vertex by a fraction so
+    // that we hit pixel centers.
+    float fx = (float)x + 0.2f;
+    float fy = (float)y + 0.5f;
+    struct Vertex vert[1] = {{{fx, fy}}};
```
Comment (aghaisas) [inline src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLRenderer.m]: Why have an array of size 1?
We can have simple struct variable.

**Review by aghaisas — CHANGES_REQUESTED** [review]

File: src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLRenderer.m (hunk 1 above)
Comment (dekonoplyov) [reply src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLRenderer.m]: Done!

**Review by aghaisas — APPROVED** [review]

**Pull request closed without merging**